src/main/java/com/chat/yourway/dto/request/ContactRequestDto.java

```diff
@@ -14,9 +17,11 @@
 @Builder
 @ToString
 public class ContactRequestDto {
-
+  @Username
   private String username;
+  @ContactEmail
   private String email;
+  @Password
   private String password;
```

Comment:
Could you rename annotations for better readability (exemple: @Password -> @ValidatePassword)